### ISSUE TITLE
ensure rvd checks the ownership and permissions of its configuration file

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#71]
```sh
$ ls -la /opt/rvc/etc/rvd.json 
-rwxrwxrwx  1 nobody  nobody  307 Sep  1 15:02 /opt/rvc/etc/rvd.json
```

And rvd starts happily:
```
$ sudo ./rvd 
2017-09-01T15:35:40HKT	INFO : Main: Initializing rvd context(at rvd.c:281)
2017-09-01T15:35:40HKT	INFO : CMD: Initializing rvd command processor(at cmd.c:514)
2017-09-01T15:35:40HKT	INFO : VPN: Initializing VPN connection manager(at vpn.c:1249)
2017-09-01T15:35:40HKT	INFO : CMD: Starting rvd command processing thread(at cmd.c:404)
```

Please make sure that ```rvd.json``` is root owned and has these permissions set:
```sh
-rw-------  1 root  wheel  307 Sep  1 15:02 /opt/rvc/etc/rvd.json
```

Also make sure that ```/opt/rvc```, ```/opt/rvc/etc``` are root owned

**Comment by jjr840430** [comment]
@erikbor Can we check the permission for the only rvd.json?
It's possible to remove config by the user when the directory has leak permissions, but not possible to edit that config in this case.
Also, I think the permission of the configuration directory should be checked while performing the installation.

**Comment by jjr840430** [comment]
fixed on #74
